ui/lib/categories.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Converts a category ID to a human-readable label.
+ * Capitalizes each word and replaces hyphens with spaces.
+ *
+ * Examples:
+ * - "internet-exposed" -> "Internet Exposed"
+ * - "iam" -> "Iam"
```

Comment:
```suggestion
 * - "iam" -> "Iam" 
```
can we leave this as `IAM`?

Reply:
I think we can make a rule, something like len < 3 (or 4) -> Capitalize completely

Reply:
Something like: 
```ts
const ACRONYMS: Record<string, string> = { iam: "IAM", e3: "E3", e5: "E5" };
  export function getCategoryLabel(id: string): string {
    if (ACRONYMS[id]) return ACRONYMS[id];
    return id.split("-").map(w => w.charAt(0).toUpperCase() + w.slice(1)).join(" ");
  }
```

Reply:
@alejandrobailo I've already implemented it with some regex for other cases

Reply:
https://github.com/prowler-cloud/prowler/pull/9561/changes#diff-badcec7a291026260f7c165995ac1da15e4598d25179170b8e1d0ba1ee3c4ddc